react-time-track/src/components/input-info-project.js

```diff
@@ -2,10 +2,12 @@
 import React from "react";
 import { jsx } from "@emotion/core";
 import { navigate } from "@reach/router";
+import { useAlert } from "react-alert";
```

Comment:
Could you explain how useAlert works, please?

Reply:
It is a library which help you to show notifications how alerts with styles